src/hooks/constants/resolvers/query/constants.tsx

```diff
@@ -0,0 +1,44 @@
+import { useCallback } from 'react';
+import { ApiPromise } from '@polkadot/api';
+import { Constants, LbpConstants, Fee } from '../../../../generated/graphql';
+import { usePolkadotJsContext } from '../../../polkadotJs/usePolkadotJs';
+import { withErrorHandler } from '../../../apollo/withErrorHandler';
+import { fetchConstants } from '../../lib/fetchConstants';
+
+export const __typenameConstants: Constants['__typename'] = 'Constants';
+export const __typenameLbpConstants: LbpConstants['__typename'] =
+  'LBPConstants';
+export const __typenameFee: Fee['__typename'] = 'Fee';
+const withTypename = (constants: Constants) => ({
+  __typename: __typenameConstants,
+  lbp: {
+    __typename: __typenameLbpConstants,
```

Comment:
Please take a look at the apollo docs, in terms of how a combination of __typename+id creates a unique identifier for the apollo cache. Once you start nesting sub-resolvers, this will be important for normalisation.

Reply:
Don't forget its always __typename+id, balances should give you a good example. when you e.g. run the app and look at the cache using the apollo chrome dev tools.

Reply:
fixed